app/src/main/java/com/tinkoff/edu/test/LoanCalcTest.java

```diff
@@ -0,0 +1,16 @@
+package com.tinkoff.edu.test;
+
+import com.tinkoff.edu.app.LoanCalcController;
+
+/**
+ * Loan Calc Tests
+ */
+public class LoanCalcTest {
+    public static void main(String... args) {
+        int requestId = LoanCalcController.createRequest();
+        System.out.println("Если requestId = 1, test passed:");
+        System.out.println(" requstId = "+ requestId);
```

Comment:
Когда конкатенируем строки старайся аккуратней относится к форматированию (когда плюсик ставим, не забываем его с двух сторон отделять пробелами, так код будет опрятней)
```suggestion
        System.out.println(" requstId = " + requestId);
```

Reply:
исправила